applications/Bono_Network.md

```diff
@@ -0,0 +1,239 @@
+
+---
+
+## Development Roadmap 🔩
+
+### Overview
+**Brief**: Unlike many other projects in the crypto space, Bono aims to solve a problem first and then scale the solution as demand grows, rather than building a massive infrastructure that is not used by anyone. Our Pilot case targets over 100,000 individuals in 9 seperate legal cases for over 250 million dollars. Bono's solution will be the only way to reliably solve these complex multijurisdiction, global cases. The Following road map and its milestones are the steps that will be taken to prove our solution works.
+
+- **Total Estimated Duration**: 4 months
+- **Full-Time Equivalent (FTE)**: 3
+- **Total Costs**:  $112,000
+- **DOT %**: 50%
+
+---
+
+### Milestone 1 — Prerequisites and App Launch Preparation
+
+**Estimated Duration**: 1 month  
+**FTE**: 1.5  
+**Costs**: 10000 USDC
+
+| Number | Deliverable                   | Specification                                                                                                                                     |
+|--------|-------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0a.    | License                       | Apache 2.0                                                                                                                                         |
+| 0b.    | Documentation                 | Detailed documentation of the project's prerequisites, including legal structuring and DAO setup                                                  |
+| 0c.    | Testing and Testing Guide     | Not applicable (focuses on preparation and structuring)                                                                                           |
+| 0d.    | Docker                        | Not applicable                                                                                                                                     |
+| 1.     | DAO Legal Structuring         | Establish legal structures for the DAO to ensure compliance and protection for global participants . [More Information](https://legalnodes.com/article/choose-a-crypto-friendly-country-for-dao#does-a-dao-need-to-be-registered-) (Paid by BONO)                                              |
+| 2.     | App Launch Preparation        | Prepare for the app launch: setting up necessary infrastructure, app store registrations, and initial awareness campaigns. Finalizing Partnership contracts with firms involved in the pilot case launch                         |
+| 3.     | Awareness and Marketing Setup | Begin groundwork for awareness campaigns supporting the launch, including partnerships with marketing agencies. over 100K individuals will be made aware. Details sent in private                                   |
+
+---
+
+### Milestone 2 — MVP Testing and Integration
+
+**Estimated Duration**: 1.5 months  
+**FTE**: 2  
+**Costs**: 54000 USDC
```

Comment:
Here's a refined version that maintains the detail and tone:

---

There are two key issues to address. First, we’re dealing with the same recurring typo, and though this may take longer than expected, I’ll push to resolve it within two months. Second, implementing this system involves multiple layers of complexity. One of the most challenging is the KYC process. As detailed in the documents, Bono operates with two main interfaces: a user-facing side and a High Table side, where lawyers decide which cases to post publicly. For the latter, we need to issue KYC-enabled wallets and LEIs for law firms accessing the platform, ensuring verification is accurate and error-free.

To achieve this, we could use a decentralized KYC provider, like Worldcoin’s ID layer, Concordium's wallet, or similar, which would enable access through a set of whitelisted wallets, allowing firms to access the High Table admin board. This complexity is essential to ensure global functionality, even in restricted regions like Iran, North Korea, Russia, Macedonia, and parts of sub-Saharan Africa. With this system, we aim to establish a baseline for legal accessibility worldwide, similar to how Interpol’s i/24 network operates in difficult jurisdictions.

Regarding costs, the $54,000 allocation includes around $1,500 per law firm for LEIs for the initial 10 firms ( this is absolutely essential in making sure that our permissioned blockchain solution is regulated and works ), the bridging or oracle solution, UI/UX to make it functional. Partnering with a provider like Concordium could lower this, though it may require additional paperwork to accommodate riskier jurisdictions ( or we could postpone these cases, something I don't want). Alternatively, we could postpone this layer, running a simpler MVP for the initial 10 law firms to validate the concept, and implement this comprehensive KYC later with VC funding. However, our primary focus remains on moving closer to our goal of setting a global standard for legal accessibility.

If the foundation is not certain about this, the dedicated budget can be spent by the foundation itself through collaboration with bono to make sure the solution is built and works in due time.

Reply:
Our grants program only focuses on the development part, so I would argue that we probably can't fund the procurement of LEIs for 10 firms. 

For the KYC solution, have you considered using [Deloitte](https://www2.deloitte.com/ch/en/pages/press-releases/articles/deloitte-integrates-kilt-identity-blockchain-creating-new-markets-with-reusable-digital-credentials.html) (generated on [Kilt](https://www.kilt.io/enterprise/deloitte) identity infrastructure)? There are currently some privacy concerns when it comes to Worldcoin.

Additionally, I understand that a permissioned chain may be the only way to achieve this, but I'm not sure if we are the best avenue for funding it.